### ISSUE TITLE
Regression: Can't set hot option to false in react preset

### ISSUE THREAD

**Issue by eirikurn** [mozilla-neutrino/neutrino-dev#576]
It runs [this path](https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/react/index.js#L35):

```
[...(options.hot ? [require.resolve('react-hot-loader/babel')] : null)]
```

Which triggers a `TypeError: Cannot read property 'Symbol(Symbol.iterator)' of null`. That null should be `[]`.

**Comment by eliperelman** [comment]
Good catch!

**Comment by eliperelman** [comment]
Released in v8.0.1.
